dependencies.yaml

```diff
@@ -219,8 +220,12 @@ dependencies:
       - output_types: [conda, requirements, pyproject]
         packages:
           - cuda-python>=11.7.1,<12.0
-          - cudf==23.8.*
           - dask>=2022.05.2
           - distributed>=2022.05.2
           - pytest
           - pytest-cov
+  notebooks:
+    common:
+      - output_types: conda
+        packages:
+          - cudf==23.8.*
```

Comment:
Add the fix for `update-version.sh` from #224.

Reply:
There isn't actually an update-version.sh fix in #224 since cudf was added back as a dependency, but it looks like I can at least remove some of it so I'll do that now.

Reply:
My mistake. I misread a0a7b596a99a99c34b85ac69acbaed6999b5ea94, you were adding it _back_ after removing it.